about_us.html

```diff
@@ -4,17 +4,85 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>About Us</title>
+    <title>Our Team</title>
+    <link rel="stylesheet" href="./about_us.css" />
     <link
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
       rel="stylesheet"
       integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
       crossorigin="anonymous"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link href="main.css" rel="stylesheet" />
   </head>
   <body>
-    <h1>About Us</h1>
+    <div
+      id="main"
+      class="container-fluid d-flex flex-column justify-content-start align-items-center"
+    >
+      <p class="h1 p-5">Our Team</p>
+      <div
+        id="cards"
+        class="container-fluid d-flex align-items-center justify-content-evenly flex-wrap m-5"
+      >
+        <div class="card-box p-5">
+          <div class="card-image">
+            <img src="./imgs/jagoda.png" alt="Jagoda Wągiel" />
+          </div>
+          <div class="card-title text-center pt-3">
+            <p class="h2">Jagoda Wągiel</p>
+          </div>
+          <div
+            class="card-role-box d-flex align-items-center justify-content-center"
+          >
+            <div
+              class="card-role p-3 rounded opacity-50 d-flex align-items-center justify-content-center position-relative"
+            ></div>
+            <div class="card-role-text text-center position-absolute">ROLA</div>
```

Comment:
po co tu jest ` position-absolute`???

Reply:
> po co tu jest ` position-absolute`???

Bo wyzej jest relative i to ta ramka kolorowa bo ona jest tak zrobiona że to element o x px szerokości za tekstem